frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -116,7 +119,8 @@ const BaseGroupExpandedComponent: React.FunctionComponent<BaseGroupExpandedProps
   labelIconClass,
   labelIcon,
   labelIconPadding,
-  onCollapseChange
+  onCollapseChange,
+  isFocused
```

Comment:
```suggestion
  // Customizations 
  isFocused,
  isHighlighted,
  IsUnhighlighted
```

Also, remove the `data` variable below and change the usage below of `!!data.isUnhighlighted` to `isUnhighlighted`.